user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Version.md

```diff
@@ -0,0 +1,65 @@
+---
+uid: Tutorial_Register_Catalog_Version
+---
+
+# Registering a new version of a connector in the Catalog
+
+This tutorial demonstrates how to add a new version to a catalog item using the [Catalog API](xref:Register_Catalog_Item).  
+We will be registering our own version of the following example [connector](https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom), so go ahead and download the solution.
+
+## Prerequisites
+
+- Organization key with permission "Register catalog items".
+- A registered catalog item, see [register a catalog item tutorial](xref:Tutorial_Register_Catalog_Item)
+
+## Overview
+
+- [Step 1: Register catalog version URL](#step-1-register-catalog-version-url)
+- [Step 2: Authentication header](#step-2-authentication-header)
+- [Step 3: Register catalog version body](#step-3-register-catalog-version-body)
+- [Step 4: Register](#step-4-register)
+
+## Step 1: Register Catalog version URL
+
+In order to register our first version, we will use the catalog API Register version call.
+Create a new Http request using the POST http method and url  <https://api.dataminer.services/api/key-catalog/v1-0/catalog/{catalogid}/version/register>
+
+![Register version http url](~/user-guide/images/tutorial_catalog_registration_version_url.png)
+
+>[!note]
+Make sure you use the same id as you used for the [catalog item registration](xref:Tutorial_Register_Catalog_Item).
+
+## Step 2: Authentication Header
+
+Add the previously obtained organization key in the **Ocp-Apim-Subscription-Key** header.
+
+![Register version http header](~/user-guide/images/tutorial_catalog_registration_version_headers.png)
+
+## Step 3: Register Catalog version Body
+
+The body of the version registration request requires a file containing our connector, for this we will be using the 
+[Skyline.DataMiner.CICD.Tools.Packager NuGet](https://www.nuget.org/packages/Skyline.DataMiner.CICD.Tools.Packager#readme-body-tab) to create a .dmprotocol package from our solution.
+
+Navigate to [Example Rates connector](https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom) and download the solution.
+
+Open a terminal window and run below command to install the packager.
+
+```powershell
+dotnet tool install --global Skyline.DataMiner.CICD.Tools.Packager --version 2.0.3
+```
+
+Run the following command to create a .dmprotocol package, adapt the directory paths as needed.
+
+```powershell
+dataminer-package-create dmprotocol "C:\Tutorials\Catalog Registration\SLC-C-Example_Rates-Custom-1.0.1.X" --name catalog_registration_tutorial --output "C:\Tutorials\Catalog Registration\Packages"
+```
+
+Add the file to the body as seen below and define versionNumber and versionDescription as well in the form-data
```

Comment:
```suggestion
Add the file to the body as seen below and define **versionNumber** and **versionDescription** as well in the **multipart/form-data**
```

Aligned it with the API docs: 
![image](https://github.com/user-attachments/assets/351d4f1c-fba7-4110-bafd-4681f62d091b)
http://localhost:8080/user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.html

However I feel like here we shuffling steps a bit. Perhaps it's more clear to say: 'Add the file to the body as multipart/form-data' or something. Similar for the versionNumber and versionDescription. I would document these as explicit steps with bullet points so I can check them off as I do them mentally. Right now it's 3 things in 1 sentence.
The screenshot currently shows the 3 fields but the content type is set to Auto, yet here we specify the form data explicitly. I am personally confused on what is expected of me as a user here.

Reply:
i reformulated it. i originally didn't want to just do the same as it was described in the user guide with the bullet points. Changed it now.

Reply:
ok